### ISSUE TITLE
[Issue] waydroid show-full-ui doesn't launch GUI

### ISSUE THREAD

**Issue by IceAsteroid** [waydroid/waydroid#864]
**Describe the bug**
while waydroid-container is running, executing "waydroid show-full-ui" doesn't launch anything.

**Screenshots**

**General information (please complete the following information):**
 - Waydroid tools Version 1.3.4
 - Waydroid Images Version `system_datetime`/`vendor_datetime` from `/var/lib/waydroid/waydroid.cfg`
```
[waydroid]
arch = x86_64
vendor_type = MAINLINE
system_datetime = 1682140641
vendor_datetime = 1682162466
suspend_action = freeze
images_path = /var/lib/waydroid/images
system_ota = https://ota.waydro.id/system/lineage/waydroid_x86_64/VANILLA.json
vendor_ota = https://ota.waydro.id/vendor/waydroid_x86_64/MAINLINE.json
binder = anbox-binder
vndbinder = anbox-vndbinder
hwbinder = anbox-hwbinder
binder_protocol = aidl3
service_manager_protocol = aidl3

[properties]
```

**Desktop (please complete the following information):**
 - OS: Void Linux rolling
 - GPU: Intel Alder Lake-P
 - Kernel version: Void Linux kernel 6.1.25_1
 - Host mesa version: mesa-22.3.5_2
 - Desktop: Sway version 1.8.1

**Smartphone (please complete the following information):**
 - Device: [e.g. Pixel 3a]
 - OS: [e.g. Ubuntu touch OTA19]
 - Mainline or Halium [e.g. Halium 9]
 - Kernel version: [e.g. 4.14.x]

**Additional context**

**Logs (please upload as file)**
 - Prop file, located at `/var/lib/waydroid/waydroid.prop`. In case that doesn't exist, use `/var/lib/waydroid/waydroid_base.prop` instead
 - Log file, located at `/var/lib/waydroid/waydroid.log`
 - *Full* (not partial) output of `waydroid logcat` while Waydroid is running
 - Output of `dmesg`

**Comment by IceAsteroid** [comment]
The sizes of the logs from "waydroid logcat" and "dmesg" are too long, should I post them here, or on another website and link them here?

**Comment by atmouse-** [comment]
> lxc-start: waydroid: ../src/lxc/utils.c: safe_mount: 1221 No such file or directory - Failed to mount "/dev/ashmem" onto "/var/lxc/containers/dev/ashmem"

Remove this line in /var/lib/waydroid/lxc/waydroid/config_nodes

**Comment by Efreak** [comment]
> The sizes of the logs from "waydroid logcat" and "dmesg" are too long, should I post them here, or on another website and link them here?

Not associated with this project, but came here from an external link. You can do one of three things:

1. Post logs on an external website
2. Post logs on https://gist.github.com (mini git repos, you can easily use it as a pastebin from the browser)
3. Post logs directly in your issue.
    - post longer logs in details tags to avoid polluting the issue
    - post then as code blocks (which you've done above) to avoid formatting issues
    - even with code blocks and details tags, a gist is better when you've got many (more than ~15, imo) lines, because you can link to a specific line if you want to (just click the line number after you submit the gist, and copy the URL). This allows you to post a full length log, but direct attention to a specific error
4. Please don't post pictures of errors. They're not searchable for other people who have the same problem. If you do post a picture, you should also post the text.

There's an example of 3 below:

-----

<details>
<summary>an example of a details tag, the source is in the other details tag below</summary>

Note that the summary tag above is *required*. Formatting inside the summary tag will not work, and links will not be converted. You can see the summary tag in the other details tag, into which I'm copying this entire post

```
This is a multiline code block
It's wrapped in triple backticks
It's useful for *short* logs
I personally use it for up to 10 lines
Unlike inline code blocks from single backticks, it provides the viewer a button to click to copy the contents
That makes it very useful for even individual commands that the user can now paste into a terminal
```

    ```
    This is another way to create a code block.
    Maybe you want the triple backticks to be *inside* the code block
    In this case, create the code block by indenting every line with 4 spaces
    I'll use this method to make the backticks available in the other details tag
    ```
</details>

<details>
<summary>the source of the section above as a code block so you can see how I did it</summary>

    <details>
    <summary>an example of a details tag, the source is in the other details tag below</summary>
    
    Note that the summary tag above is *required*. Formatting inside the summary tag will not work, and links will not be converted. You can see the summary tag in the other details tag, into which I'm copying this entire post
    
    ```
    This is a multiline code block
    It's wrapped in triple backticks
    It's useful for *short* logs
    I personally use it for up to 10 lines
    Unlike inline code blocks from single backticks, it provides the viewer a button to click to copy the contents
    That makes it very useful for even individual commands that the user can now paste into a terminal
    ```
    
        ```
        This is another way to create a code block.
        Maybe you want the triple backticks to be *inside* the code block
        In this case, create the code block by indenting every line with 4 spaces
        I'll use this method to make the backticks available in the other details tag
        ```
    </details>
</details>

I should note that an of these things are publicly documented on GitHub, it's just a matter of having to know the right terms to find them, not realizing that it's even *possible* to do such a thing on such a website.
@waydroid feel free to mark this as noncontributory or whatever GitHub allows (I've never used the feature, but I've seen comments get hidden before 🤷)

**Comment by valcarcexyz** [comment]
Do not know if related, but uninstalling `xdg-desktop-portal-gnome` solved the issue in my case

**Comment by xXRealXxx** [comment]
> Do not know if related, but uninstalling `xdg-desktop-portal-gnome` solved the issue in my case

Thanks, this for some reason fixed my issue of waydroid not starting after rebooting, even though it worked fine before a reboot on Linux Mint 21.1 with Gnome DE installed.

**Comment by IceAsteroid** [comment]
> > Do not know if related, but uninstalling `xdg-desktop-portal-gnome` solved the issue in my case
> 
> Thanks, this for some reason fixed my issue of waydroid not starting after rebooting, even though it worked fine before a reboot on Linux Mint 21.1 with Gnome DE installed.

Yes, it's related to some dependencies I guess. In gnome, waydroid can be launched without any problem, whereas, if I switch back to sway, I cannot launch it at all.

Not sure, if it's specifically a dependency issue, since, I solely tried gnome, uninstalled it, and then switch back to sway.

Since, I had tried to figure this issue out for 2 mouths, and still no success, so when trying out gnome, I still have no much attention to waydroid.

I'm just here to inform that the issue, is sway specific. I'm not gonna make waydroid work or not, at least for now, since, I have other critical troubles in the moment of my life.

Sorry for this undetailed feedback.

**Comment by gmicroul** [comment]
lxc-start: waydroid: ../src/lxc/utils.c: safe_mount: 1220 Not a directory - Failed to mount "/dev/video" onto "/usr/lib64/lxc/rootfs/dev/video"
lxc-start: waydroid: ../src/lxc/utils.c: open_without_symlink: 1084 Too many levels of symbolic links - vibrator in /usr/lib64/lxc/rootfs/sys/class/leds/vibrator was a symbolic link!


In SailfishOS, waydriod shell can open but GUI not show. how to fix?
